### PR TITLE
Re-build Audio Slot Machine (ASM)

### PR DESCRIPTION
**Pull request by meadoge**
- Re-arrange the directory tree
- ASM now will work properly with Jackpot rule _only win when 3 numbers are the same_
- Added ASCII text, cooler reason

### CODE REVIEW

File: ASM/ASM_Declaration.au3 [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+;~ Debug Mode Constants
+Global Const $DEBUG = False
+Global Const $DEBUG_WIN_VALUE = 4
+
+;~ Constants
+Global Const $COLOUR_RED   = 0xFF0000
+Global Const $COLOUR_BLACK = 0x000000
+Global Const $MIN_EDGE     = 1
+Global Const $MAX_EDGE     = 9
+
```
Comment (trhgquan) [inline ASM/ASM_Declaration.au3:34]: This spare line should be deleted.

**Review by trhgquan — CHANGES_REQUESTED** [review]
We will have some problems in reviewing. Let me do some configuration and you'll submit another pull request. Thanks!

File: ASM/ASM_Declaration.au3 (hunk 1 above)
Comment (meadoge) [reply ASM/ASM_Declaration.au3:34]: Got it.

**Review by meadoge — COMMENTED** [review]
I'll fix it and wait.

**Pull request closed without merging**